docs/md_docs/flux_examples.md

```diff
@@ -0,0 +1,273 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.2'
+      jupytext_version: 1.7.1
+  kernelspec:
+    display_name: Python 3
+    language: python
+    name: python3
+---
+
+<!-- #region -->
+# Point Source Fluxes and Multiple Sources
+
+Once one has computed a spectral to a point source, getting the flux of that source is possible.
```

Comment:
Nitpicky, but we can get (spatially integrated) fluxes from extended sources as well.

Reply:
should we make a link to some docs on that?

Reply:
Or, just change point source to "point source and extended source" for now :D